com.unity.netcode.gameobjects/Tests/Runtime/NetworkTransform/NetworkTransformTests.cs

```diff
@@ -45,7 +41,8 @@ public override IEnumerator Setup()
             m_ClientSideClientPlayer = clientClientPlayerResult.Result;
         }
 
-        [UnityTest]
+        // TODO: rewrite after perms & authority changes
```

Comment:
we have to revisit these tests after permissions/authority changes coming up — for now, commented-out.